docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>I think this should be "Solving" and not "Deriving"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>It is actually both... we derive partial derivative, and then at the end, we try to solve it when we replace left-hand side with 0.</p><p> </p><p>But, as you suggested, I think "Solving" is better as it is what we are doing at the end of the day. I will change it in the next set of commits. </p>